Riot/Modules/Room/CellData/RoomBubbleCellData.m

```diff
@@ -107,7 +107,7 @@ - (instancetype)initWithEvent:(MXEvent *)event andRoomState:(MXRoomState *)roomS
                 }
                 else
                 {
-                    self.tag = RoomBubbleCellDataTagRoomCreateConfiguration;
+                    self.tag = RoomBubbleCellDataTagRoomCreationIntro;
```

Comment:
I believe the original value of `RoomBubbleCellDataTagRoomCreateConfiguration` was unintentional, and resulted in the need for the insert / remove dance in `RoomDataSource`: a whole bunch of events in a timeline can have the `CreateConfiguration` flag.